index.html

```diff
@@ -0,0 +1,453 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
```

Comment:
For this section, just as the previous reviewer pointed out, please make your hamburger effective and functional as stated in the project requirements. This is because when on mobile view, your users won't be able to navigate your web app. That will go against the good practice for web assessability 😮‍💨  
Kindy fix it and give your project a nice accessibility to tally with your good job thus far 👍  
  ## Here are the requirements for your review and implementation 
  
![chhhh](https://user-images.githubusercontent.com/50990925/211042222-b706c563-8121-497e-8331-ed645634cb92.png)